google-home-community.groovy

```diff
@@ -562,6 +578,112 @@ private deviceTraitPreferences_Dock(deviceTrait) {
     }
 }
 
+@SuppressWarnings(['UnusedPrivateMethod', 'MethodSize'])
+private deviceTraitPreferences_EnergyStorage(deviceTrait) {
+    googleEnergyStorageDistanceUnitForUX = [
+        "KILOMETERS":      "Kilometers",
+        "MILES":           "Miles",
+    ]
+    googleCapacityUnits = [
+        "SECONDS":         "Seconds",
+        "MILES":           "Miles",
+        "KILOMETERS":      "Kilometers",
+        "PERCENTAGE":      "Percentage",
+        "KILOWATT_HOURS":  "Kilowatt Hours",
+    ]
+    section("Energy Storage Settings") {
+        input(
+            name: "${deviceTrait.name}.isRechargeable",
+            title: "Rechargeable",
+            type: "bool",
+            defaultValue: false,
+            required: true,
+            submitOnChange: true
+        )
+        input(
+            name: "${deviceTrait.name}.queryOnlyEnergyStorage",
+            title: "Query Only Energy Storage",
+            type: "bool",
+            defaultValue: false,
+            required: true,
+            submitOnChange: true
+        )
```

Comment:
Should this default to `true`?  The vast majority of devices probably don't have a dedicated charge command.

Reply:
I was matching what Google was expecting as default, but agreed, true makes more sense.